repository/src/main/java/ca/bc/gov/repository/bcsc/BcscAuthExtension.kt

```diff
@@ -19,10 +13,10 @@ import kotlin.coroutines.resumeWithException
 * Created by amit_metri on 31,January,2022
 */
 
-suspend fun awaitFetchFromIssuer(uri: Uri): AuthorizationServiceConfiguration =
+suspend fun awaitFetchFromIssuer(uri: Uri): net.openid.appauth.AuthorizationServiceConfiguration =
     suspendCancellableCoroutine { continuation ->
 
-        AuthorizationServiceConfiguration.fetchFromIssuer(
+        net.openid.appauth.AuthorizationServiceConfiguration.fetchFromIssuer(
```

Comment:
Is it possible to declare the imports in the top for  AuthorizationServiceConfiguration, etc?

Reply:
updated